docs/framework/windows-workflow-foundation/wfc-exe-workflow-compiler-tool.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "Wfc.exe (Workflow Command-line Compiler Tool)"
+description: Understand wfc.exe, the Workflow command line compiler tool. 
+ms.date: "10/10/2020"
+helpviewer_keywords: 
+  - "wfc [Workflow]"
+  - "compiler tool"
+  - "wfc.exe"
+  - "Workflow, compilation"
+  - "Workflow, XOML files"
+  - "Workflow, wcf"
+---
+# wfc.exe (Workflow Command-line Compiler Tool)
+> [!NOTE]
+> [!INCLUDE[DeprecatedContent](deprecatedcontent-md.md)]  
+
+The wfc.exe workflow command-line compiler tool works with old workflow markup files that have the file extension *.xoml* (obsoleted).
+
+## Compilation process
+
+When workflows are compiled, the following procedures are performed as part of the compilation process:
+
+- Validation is performed to ensure that the workflow activities validate based on the rules that the activities have set for themselves. If there are validation errors, the compiler returns a list of the errors.  
+- A partial class is generated from the markup definition that is input into the compiler.  
+
+- Code is generated to help with the run-time execution of the activities. Event subscriptions are generated, which help activities know when the activities they contain are finished executing.  
+- The partial classes generated from the markup file and the partial classes from the code file are entered into the .NET Framework C# or Visual Basic compiler. The output of this process is the .NET assembly, WorkflowSample.dll. This can be deployed to run the workflow.
+
+### Compiler options
+
+This section shows the options for the wfc.exe workflow command-line compiler.
+
+    Microsoft (R) Windows Workflow Compiler version 3.0.0.0
```

Comment:
This lint error will need to be fixed. I suggest code fencing the entire output section and removing the indentation.

\```output
    Microsoft (R) Windows Workflow Compiler version 3.0.0.0
    ...
\```

Reply:
fix in next commit